src/reporting/reportingClient.ts

```diff
@@ -100,16 +292,46 @@ export class ReportingClient {
     );
   }
 
+  /**
+   * ---    Gets all Report Mappings for a Report.    ### Authentication    Requires `Authorization` header with valid Bearer token for scope `insights:read`.    For more documentation on authorization and how to get access token visit [OAUTH2 Authorization](https://developer.bentley.com/apis/overview/authorization/) page.    ### Authorization    User must have `insights_view` permission(s) assigned at the Project level. iModel specific permissions may also be applied at the iModel level if iModel level permissions are enabled.    Alternatively the user should be an Organization Administrator for the Organization that owns a given Project or iModel.    An Organization Administrator must have at least one of the following roles assigned in User Management: Account Administrator, Co-Administrator, or CONNECT Services Administrator. For more information about User Management please visit our Bentley Communities [Licensing, Cloud, and Web Services](https://communities.bentley.com/communities/other_communities/licensing_cloud_and_web_services/w/wiki/50711/user-management-2-0) wiki page.    ### Rate limits    All iTwin Platform API operations have a rate limit. For more documentation on that visit [Rate limits and quotas](https://developer.bentley.com/apis/overview/rate-limits/) page.    ---
+   * @summary Get Report Mappings
+   * @param {string} reportId The Report Id.
+   * @param {string} accessToken OAuth access token with scope &#x60;insights:read&#x60;
+   * @memberof ReportingClient
+   */
   public async getReportMappings(accessToken: AccessToken, reportId: string) {
-    return this._reportsApi.getReportMappings(
-      reportId,
-      accessToken,
-      undefined,
-      undefined,
-      ACCEPT
-    );
+    const reportMappings: Array<ReportMapping> = [];
+
+    let response: ReportMappingCollection;
+    let continuationToken: string | undefined;
+
+    do {
+      response = await this._reportsApi.getReportMappings(
+        reportId,
+        accessToken,
+        undefined,
```

Comment:
Do we want to support top?

Reply:
yes, but not right now
leaving at minimum implementation and we will fix the client properly soon